style.css

```diff
@@ -440,3 +453,476 @@ li {
+    background-color: #2230d2;
+  }
+
+  .blue-button {
+    color: #fff;
+    background: #4053fc;
+    text-decoration: none;
+    padding: 1vh;
+    border: 1px solid #7f8cff;
+    border-radius: 8px;
+    font-size: 1.5vh;
+    font-weight: 500;
+    width: 7vw;
+    margin-top: 2vh;
+  }
+
+  .about-sec {
+    display: flex;
+    flex-direction: row;
+    background-color: #fff;
+    border-top-right-radius: 9vh;
+    margin-top: 10vh;
+    height: 60vh;
+  }
+
+  .about-content-container {
+    display: flex;
+    flex-direction: column;
+    width: 40vw;
+    padding-left: 10vw;
+  }
+
+  .about-intro {
+    color: #091e42;
+    font-size: 4vh;
+    font-weight: bold;
+    font-style: normal;
+    line-height: 5vh;
+  }
+
+  .about-description {
+    font-size: 1.5vh;
+    font-weight: normal;
+    font-style: normal;
+    color: #344563;
+    margin-top: 1.5vh;
+    width: 33vw;
+  }
+
+  .home-connect {
+    font-weight: 500;
+    font-style: normal;
+    font-size: 1.5vh;
+    color: #7f8cff;
+    margin-top: 1vh;
+    padding-left: 15vw;
+    width: 10vw;
+  }
+
+  .about-button {
+    color: #396df2;
+    text-decoration: none;
+    padding: 1.5vh;
+    border: 1px solid #607dff;
+    border-radius: 8px;
+    font-style: normal;
+    font-size: 1.6vh;
+    font-weight: 500;
+  }
+
+  .about-icon {
+    padding: 0;
+  }
+
+  .button-one {
+    margin-top: 3vh;
+  }
+
+  .about-button:hover {
+    color: #fff;
+    background-color: #6070ff;
+  }
+
+  .about-button:focus {
+    color: #fff;
+    background-color: #2230d2;
+  }
+
+  .about-skill {
+    padding-top: 12vh;
+  }
+
+  .tech-names-list {
+    display: flex;
+    flex-direction: column;
+    width: 40vw;
+    margin: 0;
+    height: auto;
+  }
+
+  .tech-names-list h3 {
+    font-weight: 500;
+    font-style: normal;
+    font-size: 1.6vh;
+    color: #344563;
+  }
+
+  .language-arrow,
+  .framework-arrow,
+  .skill-arrow {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  .fa-angle-down,
+  .fa-angle-right {
+    margin: 0;
+  }
+
+  .language-arrow {
+    margin-bottom: 4vh;
+    margin-top: 0;
+  }
+
+  .framework-arrow {
+    margin-top: 4vh;
+    margin-bottom: 2.5vh;
+  }
+
+  .skill-arrow {
+    margin-top: 2.5vh;
+    margin-bottom: 2.5vh;
+  }
+
+  .name-two,
+  .name-three {
+    width: 40vw;
+  }
+
+  .skill-list {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: space-between;
+    margin-top: 0;
+    margin-right: 0;
+    width: 32vw;
+  }
+
+  .skills {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    border-radius: 8px;
+    background-color: #f7f7f9;
+    height: 11vh;
+    width: 10vw;
+    margin: 0;
+  }
+
+  .skill-para {
+    padding: 0;
+    color: #253858;
+    font-size: 1vw;
+    font-weight: 500;
+  }
+
+  .skills > img {
+    padding-left: 0;
+    padding-top: 1vh;
+    padding-bottom: 1vh;
+    height: 5vh;
+    width: 5vw;
+  }
```

Comment:
- [ ] In the about section, the images look stretched because of screen-based values (`vh`). I suggest you remove those values, it will look much better.

| With 5vh on  | Without 5vh |
| ------------- | ------------- |
| ![](https://i.imgur.com/1QsLhHt.png)  | ![](https://i.imgur.com/KXXX2gm.png)  |

Reply:
Thanks for the remark. I will fix these.